src/stores/messages/SavedMessageViewType.ts

```diff
@@ -0,0 +1,32 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import { action, observable } from 'mobx';
+import { MessageViewType } from '../../models/EventMessage';
```

Comment:
I think thats good idea to delete SaveSaver and use class for managing message viewType, but i want suggest some changes:

i think it should look something like this:
```
class MessageViewStore {
   constructor(public messageId: string, public session: string) {
         reaction(() => this.displayRule) // here we can react to change of rule and change view message view type
   }
   
   @computed displayRule

   viewType // will default to displayRule.viewType

   setViewType

}

```